config/default.yml

```diff
@@ -43,6 +43,11 @@ RSpec/BeforeAfterAll:
   - spec/support/**/*.rb
   StyleGuide: http://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/BeforeAfterAll
 
+RSpec/ContextDescription:
+  Description: "`context` block descriptions should start with 'when'."
+  Enabled: false
```

Comment:
false by default because of:
```
Offenses:

spec/rubocop/cop/rspec/example_length_spec.rb:59:11: C: RSpec/ContextDescription: Start context description with when.
  context 'with CountComments enabled' do
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
spec/rubocop/cop/rspec/example_wording_spec.rb:4:11: C: RSpec/ContextDescription: Start context description with when.
  context 'with configuration' do
          ^^^^^^^^^^^^^^^^^^^^
spec/rubocop/cop/rspec/multiple_expectations_spec.rb:6:11: C: RSpec/ContextDescription: Start context description with when.
  context 'without configuration' do
          ^^^^^^^^^^^^^^^^^^^^^^^
spec/rubocop/cop/rspec/multiple_expectations_spec.rb:99:11: C: RSpec/ContextDescription: Start context description with when.
  context 'with meta data' do
          ^^^^^^^^^^^^^^^^
spec/rubocop/cop/rspec/multiple_expectations_spec.rb:135:11: C: RSpec/ContextDescription: Start context description with when.
  context 'with configuration' do
          ^^^^^^^^^^^^^^^^^^^^
spec/rubocop/cop/rspec/return_from_stub_spec.rb:8:11: C: RSpec/ContextDescription: Start context description with when.
  context 'with EnforcedStyle `and_return`' do
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
spec/rubocop/cop/rspec/return_from_stub_spec.rb:120:11: C: RSpec/ContextDescription: Start context description with when.
  context 'with EnforcedStyle `block`' do
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
spec/rubocop/cop/rspec/shared_context_spec.rb:4:11: C: RSpec/ContextDescription: Start context description with when.
  context 'shared_context' do
          ^^^^^^^^^^^^^^^^
spec/rubocop/cop/rspec/shared_context_spec.rb:58:11: C: RSpec/ContextDescription: Start context description with when.
  context 'shared_examples' do
          ^^^^^^^^^^^^^^^^^
spec/rubocop/cop/rspec/single_argument_message_chain_spec.rb:51:13: C: RSpec/ContextDescription: Start context description with when.
    context 'with single-key hash argument' do
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
spec/rubocop/cop/rspec/single_argument_message_chain_spec.rb:80:13: C: RSpec/ContextDescription: Start context description with when.
    context 'with multiple keys hash argument' do
            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
spec/rubocop/rspec/example_spec.rb:39:11: C: RSpec/ContextDescription: Start context description with when.
  context 'value object semantics' do
          ^^^^^^^^^^^^^^^^^^^^^^^^
spec/rubocop/rspec/language/selector_set_spec.rb:14:11: C: RSpec/ContextDescription: Start context description with when.
  context '#include?' do
          ^^^^^^^^^^^
spec/rubocop/rspec/language/selector_set_spec.rb:24:11: C: RSpec/ContextDescription: Start context description with when.
  context '#node_pattern' do
          ^^^^^^^^^^^^^^^
spec/rubocop/rspec/language/selector_set_spec.rb:30:11: C: RSpec/ContextDescription: Start context description with when.
  context '#node_pattern_union' do
          ^^^^^^^^^^^^^^^^^^^^^
spec/rubocop/rspec/language/selector_set_spec.rb:36:11: C: RSpec/ContextDescription: Start context description with when.
  context '#send_pattern' do
          ^^^^^^^^^^^^^^^
spec/rubocop/rspec/language/selector_set_spec.rb:42:11: C: RSpec/ContextDescription: Start context description with when.
  context '#block_pattern' do
          ^^^^^^^^^^^^^^^^

149 files inspected, 17 offenses detected
```